lib/index.js

```diff
@@ -11,6 +12,42 @@ var defaults = {
 
 exports.register = function(plugin, options, next) {
 
+  // handlers for reporting the different formats of incoming log messages
+  // this can be expanded to support an api-like interface via logging statements
+  var MessageHandlers = {
```

Comment:
The only real use case I can think of is adding a metric.  What other use cases can you think of?